common/record/_recordValidator/attributeValidator.js

```diff
@@ -42,9 +42,7 @@ const _validateNodeValidations = (survey, record, nodeDef) => async (propName, n
     if (!valid) {
       const messages = _getCustomValidationMessages(survey, expression)
 
-      errorMessage = ValidationResult.newInstance(
-        Validation.keys.customErrorMessageKey,
-        null,
+      errorMessage = ValidationResult.newInstanceCustom(
```

Comment:
why a custom constructor? I would have left as it was. otherwise for every exception we need to add a specific constructor

Reply:
I'll remove it